src/utils/aotf.js

```diff
@@ -678,7 +678,7 @@ export async function mutate (mutation, args, apolloClient) {
     }
     // command in a different format (e.g. info command)
     return [TaskState.SUBMITTED, result]
-  } catch {
+  } catch (error) {
```

Comment:
I think one of the Webpack loaders was not happy about this syntax, even though it was working before. I think a new and more strict rule was added somewhere in some dependency :man_shrugging: